.travis.yml

```diff
@@ -35,3 +35,6 @@ script:
   - python plot.py --dataset random-xs-20-angular --output plot-batch.png --batch
   - python -m unittest test/test-metrics.py
   - python create_website.py --outputdir . --scatter --latex
+  - python run.py --definitions algosP.yaml --dataset random-xs-20-angular --local --runs 20
```

Comment:
I'm not sure why this has to run as a part of every library.

Why not just make it its own separate Travis run?

But why don't I merge this and just fix it myself since this PR has been sitting open for a while